ansible/roles/ocp4-workload-mlops/tasks/post_workload.yml

```diff
@@ -0,0 +1,43 @@
+---
+# Implement your Post Workload deployment tasks here
+- name: output workshop info
+  debug:
+    msg:
+    - "agnosticd_user_info: "
+    - "agnosticd_user_info: Provisioned for {{ num_users }} user(s)"
```

Comment:
Usage of `agnosticd_user_info` is as a module to replace `debug`. This should be:

```
- name: output workshop info
  agnosticd_user_info:
    msg:
    - ""
    - "Provisioned for {{ num_users }} user(s)"
```

Reply:
Updated